pixel/head.html

```diff
@@ -0,0 +1,20 @@
+<script>
+  partytown = {
+    forward: ['dataLayer.push', 'vtexrca'].concat("{{settings.forward}}".split(',')),
+    lib: '/_v/public/assets/v1/published/vtex.partytown@0.0.1-alpha.10/public/~partytown/',
+    resolveUrl: function (url) {
+      var domain = url.hostname.replace('www.', '')
+
+      if ("{{settings.toProxyDomains}}".split(',').indexOf(domain) >= 0) {
+        // TODO: This URL is just for test purposes and must be removed before deploying
+        return new URL(`https://reverse-proxy-vtex.herokuapp.com/${url}`);
```

Comment:
Idk if it's okay to use this external domain.

Reply:
I'll keep it while we don't deploy for testing purposes

I'm waiting for the SRE to give us a feedback on this topic